DESCRIPTION

```diff
@@ -0,0 +1,11 @@
+Package: geomstats

```

Comment:
Once we merge PR #2 , you should 'rebase' your branch on r-geomstats master.

https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request